### ISSUE TITLE
One feedback about the SnapLinksPlus_v3.1.1b1

### ISSUE THREAD

**Issue by WagnerGMD** [cpriest/SnapLinksPlus#143]
Hello,

I don't know why neither when but **apparently**, Mozilla has changed the process for the login... Right now, after several attempts, the login still refuse to work (seem to rely on https://accounts.firefox.com) ...  (In the case, yes the LocalStorage and the cookie are enable but it seem's, there's a wrong detection about the both... the form wasn't loaded.)

Then, finally, I decide to post my review [here (SnapLinksPlus GitHub)](https://github.com/cpriest/SnapLinksPlus/). So, this first beta (v3.1.1) is far better than the v3.1.0.4 (and also the v3.0.0.2 available on [this page).](https://addons.mozilla.org/en-US/firefox/addon/SnapLinksPlus/).
Because, it's very simple, I'm glad to had discover it (via the [(SnapLinksPlus) release page](https://github.com/cpriest/SnapLinksPlus/releases) and not the Mozilla Addon website) and to confirm. Indeed, the feature which allow to recopy all the links in the clipboard (via the keyboard **ctrl**) has been restored. I had expected this feature since a long time (like when the v3.0.0.2 has been published).
About the "Greatest Font Size", (on the [SnapLinksPlus welcome page](http://cpriest.github.io/SnapLinksPlus/welcome.html), the keyboard **alt** seem's to not work. I would rather to open the both links (and not only one of them).

About the v3.1.0.4, I didn't understand. For me, it's a regression... Even the v3.0.0.2 is better because SnapLinksPlus has opened each links in one new tab(s). Basically, only one tab was open with the v3.1.0.4.

In the futur, I hope more features will be available such as the previous one which allowed to avoid to reopen another tab(s) for the same link(s).

Best regards.

**Comment by cpriest** [comment]
I'm really not sure what your asking for/about here... could you summarize the issue?  Each issue should be in its own ticket, please also be sure there is not already a ticket open about a given issue.

**Comment by WagnerGMD** [comment]
> About the "Greatest Font Size", (on the SnapLinksPlus welcome page, the keyboard alt seem's to not work. I would rather to open the both links (and not only one of them).

Pardon, so there's the issue and I didn't know neither find if one ticket already exist...

Apparently, the v3.1.1 was released five days ago. Then, if you wish you can also recopy this feedback (or add one little note, or one link, etc) here :  https://addons.mozilla.org/en-US/firefox/addon/SnapLinksPlus/ because the people **can miss** the information. Where is the beta version ? From my point of view, it isn't clear and I believe, it's might be true (missed) for the users like tucuxia (or tina21, etc) because each review(s) was posted only a few days ago.

**Comment by cpriest** [comment]
I decided to just do the next point release, but it is waiting for approval.  It should come out today and will be v3.1.1.  I could not create an animated gif of the right-drag and hold down alt to select all links (ignoring greatest font size).  Perhaps it's not working as expected.  You need to start a drag (with a regular right click drag, then hold down alt while dragging).  

I will try to update the info on the Snap Links addon page timed more appropriately with the actual approval of the add-on date.
